estela-web/webpack.mf.config.js

```diff
@@ -62,7 +62,7 @@ module.exports = (env) => {
                 name: "estela_web",
                 filename: "remoteEntry.js",
                 remotes: {
-                    ExternalDropdownComponent: `user_dropdown_module@${env.remoteURL}remoteEntry.js`,
+                    BillingModule: `user_dropdown_module@${env.remoteURL}remoteEntry.js`,
```

Comment:
Use a generic name instead of **BillingModule**

Reply:
Ok changed this to ExternalComponent

Reply:
Changed it to ExternalComponent